ceremonies/2020/r3-key.yaml

```diff
@@ -2,10 +2,11 @@ ceremony-type: key
 pkcs11:
   module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
   pin: 1234
-  store-key-in-slot: 703725468
+  store-key-in-slot: 732394342
   store-key-with-label: int-r3
 key:
   type: rsa
   rsa-mod-length: 2048
 outputs:
-  public-key-path: int-r3.key.pem
+  public-key-path: ./int-r3.key.pem
+  
```

Comment:
I still don't understand what's going on here. Why does this file have different line endings than the others?

Reply:
That might have been a leftover from temporarily editing the file in vim and then opening it in vscode, maybe an indent spacing issue, maybe solarflares.